connexta/ddf-registry
# DDF-1964

#### What does this PR do?

This PR adds new functionality for the Source UI. This includes changes to the way sources are viewed on the main source page, adding accordion menus to the source modal popup, changing of various terms to better reflect their intended function, changes to the way sources are saved to the backend, and allowing users to publish nodes to configured registries.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@mcalcote @vinamartin @gordocanchola @ricklarsen @clockard 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
#### How should this be tested?

Install securely (dev=true) and with the correct hostname configured. Test federating with other users running the same build. Make sure nodes are added correctly with their corresponding values being saved. Check that nodes can be published to various registries and also seen from those registries.
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Checklist:
